src/zimlet/com_zimbra_date/com_zimbra_date_pt_BR.properties

```diff
@@ -1,17 +1,17 @@
 #
 # ***** BEGIN LICENSE BLOCK *****
-# Zimbra Collaboration Suite Zimlets
-# Copyright (C) 2009, 2010, 2011, 2012, 2013, 2014, 2015, 2016 Synacor, Inc.
-#
-# This program is free software: you can redistribute it and/or modify it under
-# the terms of the GNU General Public License as published by the Free Software Foundation,
-# version 2 of the License.
-#
-# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
-# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-# See the GNU General Public License for more details.
-# You should have received a copy of the GNU General Public License along with this program.
-# If not, see <https://www.gnu.org/licenses/>.
+# Zimbra Collaboration Suite Zimlets

```

Comment:
this looks changed, but couldn't notice any difference
please revert this change